### DISCUSSION TITLE
What is DII_NOETOH ?

### DISCUSSION THREAD

**Discussion by Raybee1126** [jamesjiadazhan/dietaryindex#210] [Q&A] [answered]
When calculating DII, I get DII_ALL and DII_NOETOH, but I don't know what is DII_NOETOH. Sincerely looking for help

**Comment by jamesjiadazhan** [comment, marked as answer]
Thanks for using dietaryindex! 

It is the total summarized DII score without the alcohol (ethanol) component score.

Let me know if you have other questions. If possible, please give our work a Star on GitHub and cite our work when you publish the work. It helps recognize and appreciate our intellectual efforts and contributions.

<img width="1387" alt="image" src="https://github.com/jamesjiadazhan/dietaryindex/assets/108076575/ee69a432-70ba-4e85-a650-7c1cbddd94ff">

Jiada James Zhan, Rebecca A Hodge, Anne Dunlop, et al. Dietaryindex: A User-Friendly and Versatile R Package for Standardizing Dietary Pattern Analysis in Epidemiological and Clinical Studies. bioRxiv. Published online August 07, 2023:2023.08.07.548466. doi:10.1101/2023.08.07.548466

**Comment by Raybee1126** [comment]
Thanks a lot! That’s really fantastic work!

From: James Jiada Zhan ***@***.***>
Date: 2023-10-10 00:46:53
To: jamesjiadazhan/dietaryindex ***@***.***>
Cc: Raybee1126 ***@***.***>,Author ***@***.***>
Subject: Re: [jamesjiadazhan/dietaryindex] What is DII_NOETOH ? (Discussion #210)

Thanks for using dietaryindex!
It is the total summarized DII score without the alcohol (ethanol) component score.
Let me know if you have other questions. If possible, please give our work a Star on GitHub and cite our work when you publish the work. It helps recognize and appreciate our intellectual efforts and contributions.
Jiada James Zhan, Rebecca A Hodge, Anne Dunlop, et al. Dietaryindex: A User-Friendly and Versatile R Package for Standardizing Dietary Pattern Analysis in Epidemiological and Clinical Studies. bioRxiv. Published online August 07, 2023:2023.08.07.548466. doi:10.1101/2023.08.07.548466
